tests/testthat/test-run.R

```diff
@@ -1,79 +1,42 @@
 context("run")
 
-## Same as in read; we generate a report and then break it
 test_that("minimal", {
   path <- prepare_orderly_example("minimal")
   on.exit(unlink(path, recursive = TRUE))
 
-  config <- orderly_config$new(path)
```

Comment:
This test was a hot mess so I've just got rid of it and replaced with a sensible minimal test